browser-bunyan/tsconfig.json

```diff
@@ -0,0 +1,20 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```

Comment:
The build you linked to is not the latest. [This is the latest Travis CI run](https://travis-ci.org/PaulLockwood/DefinitelyTyped/builds/188718862) (which passed).